src/main/java/thePackmaster/powers/bladestormpack/GaleForcePower.java

```diff
@@ -0,0 +1,64 @@
+package thePackmaster.powers.bladestormpack;
+
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.vfx.bladestormpack.GaleForceEffect;
+
+import static com.megacrit.cardcrawl.dungeons.AbstractDungeon.player;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+
+//REFS: WhirlwindAction (base game)
+public class GaleForcePower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("GaleForcePower");
+    private static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    private static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    private final int EXTRA_WINDRUSH;
+    private final int CARDS_TO_DRAW;
+    private final int DRAW_COST_THRESHOLD;
+
+    public GaleForcePower(AbstractCreature owner, int amount, int extraWindrush, int draw, int threshold) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+        EXTRA_WINDRUSH = extraWindrush;
+        CARDS_TO_DRAW = draw;
+        DRAW_COST_THRESHOLD = threshold;
+        updateDescription();
+    }
+
+    @Override
+    public void onUseCard(AbstractCard c, UseCardAction action) {
+        if (amount <= 0 || c.type != AbstractCard.CardType.ATTACK) { return; }
+
+        this.flashWithoutSound();
+        addToBot(new VFXAction(new GaleForceEffect(), 0.0F));
+
+        atb(new ApplyPowerAction(player, player, new WindrushPower(player, (c.cost + EXTRA_WINDRUSH) * amount)));
```

Comment:
It's better to use `Wiz.getLogicalCardCost` here and in the check below, so that things like X cost cards and Swivel's free attack power are handled correctly.